new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWITValidation.java

```diff
@@ -34,7 +42,21 @@ public void testCreatingMIIImage() {
 
     // build the model file list
     List<String> modelList = Collections.singletonList(MODEL_DIR + "/" + WDT_MODEL_FILE);
-
+    
+    // build an application archive using what is in resources/apps/APP_NAME
+    boolean archiveBuilt = buildAppArchive(
+        withAppParams()
```

Comment:
It feels like the use of withAppParams() is disguising what is being built and tested.  I'll look below to see what this method does, but it's not clear what these statements do.

Reply:
withAppParams() method is a way of getting an instance of params without using the impl directly in the test code. The particular one just returns a new instance.

Reply:
Will it help if I add more explanation in the javadoc?

Reply:
this feels too magical - i assume it is setting some defaults, we should either not do that, or change the name to something like `withDefaultAppParams()` so it is clear what is happening

Reply:
I'll change the method name to defaultAppParams to be a little bit more clear. this method also hide things that a test writer does not need to know, for example, the version of the installers and etc. All parameters that a test writer cares needs to be set once he gets an AppParams instance.